sunpy/visualization/animator/base.py

```diff
@@ -257,8 +257,8 @@ def _make_axes_grid(self):
             vert = [ysize, pad_size] * self.num_sliders + \
                    [large_pad_size, large_pad_size, Size.AxesY(self.axes)]
         else:
-            vert = [ysize, pad_size] * self.num_sliders + \
-                   [large_pad_size, Size.AxesY(self.axes)]
+            vert = [ysize, large_pad_size] * self.num_sliders + \
+                   [Size.AxesY(self.axes)]
```

Comment:
Deleting one of the pad_sizes is ok?

Reply:
Frankly, I need to test it a bit more.

Reply:
Well this does solve most of the problem, bu tvery satisfactorily. Also, it's specific to the example in the docs.

Reply:
How well does this fix the issue?

Reply:
Changing y-axis values no longer affects the plot size as earlier but changing the window size does affect the whole plot. Everything goes out of ratio in y-axis again and depending on how it's done, the slider comes on top of the plot.